Tests/Imagine/Cache/SignerTest.php

```diff
@@ -21,9 +22,9 @@ class SignerTest extends AbstractTest
 {
     public function testImplementsSignerInterface()
```

Comment:
This comment is not intended for action/review...

Now that I look at it, this is a very strange thing to test for!  We typehint on SignerInterface, don't we?